docs/design/architecture.md

```diff
@@ -279,12 +535,24 @@ file and device mapping mechanisms:
   share pages.
 
 Kata Containers uses the following steps to set up the DAX mappings:
-1. QEMU is configured with an NVDIMM memory device, with a memory file
-  backend to map in the host-side file into the virtual NVDIMM space.
+
+1. QEMU is configured with an NVDIMM memory device which has a memory file
+   backend to map in the host-side file into the virtual NVDIMM space.
 2. The guest kernel command line mounts this NVDIMM device with the DAX
-  feature enabled, allowing direct page mapping and access, thus bypassing the
-  guest page cache.
+   feature enabled, allowing direct page mapping and access, thus bypassing the
+   guest page cache.
 
 ![DAX](arch-images/DAX.png)
 
-Information on the use of NVDIMM via QEMU is available in the [QEMU source code](http://git.qemu-project.org/?p=qemu.git;a=blob;f=docs/nvdimm.txt;hb=HEAD)
+Information on the use of NVDIMM via QEMU is available in the [QEMU
+source code](http://git.qemu-project.org/?p=qemu.git;a=blob;f=docs/nvdimm.txt;hb=HEAD)
```

Comment:
Better point to QEMU's github page.

Reply:
I've pointed to the official site as that seems safer since the GH page is a clone (albeit an official one).